site/documentation/content/admin-guide/mongodb-device-registry-config.md

```diff
@@ -128,3 +128,29 @@ The file needs to contain at least all versions of the key that values in the da
 Otherwise the registry will not be able to decrypt these values during reading. The `defaultKey` property indicates the
 version of the key that should be used when encrypting values. Please refer to the
 [CryptVault project](https://github.com/bolcom/cryptvault) for additional information regarding key rotation.
+
+## Messaging Configuration
```

Comment:
what about the notification configuration?

Reply:
Currently the messaging configuration is used for that (in the AMQP case) - see https://github.com/eclipse/hono/pull/3019#discussion_r782191263.

Reply:
I've modified the whole `Messaging Configuration` chapter on all relevant admin guides here accordingly.